features/fixtures/mazerunner/src/main/java/com/bugsnag/android/mazerunner/scenarios/CXXAbortScenario.java

```diff
@@ -0,0 +1,30 @@
+package com.bugsnag.android.mazerunner.scenarios;
+
+import android.content.Context;
+
+import com.bugsnag.android.Configuration;
+
+import org.jetbrains.annotations.NotNull;
+
+public class CXXAbortScenario extends Scenario {
```

Comment:
I can't really foresee why it would matter. Some cases touch native bits, some touch Java bits, other touch both. We aren't actively developing it as an example of how an app should be built as its just a test fixture.

Reply:
There are now ~65 classes in this package. It feels like it would be nice to organise them in some way.